### ISSUE TITLE
setting learning rate

### ISSUE THREAD

**Issue by AllieLan** [open-mmlab/mmpretrain#1865]
### 描述该功能

when proposing a new backbone based on a resnet50 in mmpretrain, I only have 4gpus on one server, and I found from the log that resnet50 is pretrained on 8 gpus, so I wonder how to set the learning rate if I use 4 gpus to train models, and is there any relationship among learning rate, the number of gpus, and batch size. 

### 是否希望自己实现该功能？

- [ ] 我希望自己来实现这一功能，并向 MMPreTrain 贡献代码！

**Comment by Brokttv** [comment]
I'm not sure what you trying to so, hope you weren't thinking about training the model from scratch. 4 gpus are more than enough and you can use transfer learning if you have a relatively small custom dataset (>10k samples). and lr has to do with the size of data thus the batch_size. you better set a lr watcher that tweaks it incrementally as you progressing through training steps. the more the model is leaning, the higher the lr you can use to speed things up a notch.
